Classes/Middleware/StyleguideRouter.php

```diff
@@ -149,13 +151,30 @@ public function process(
                         'twoLetterIsoCode' => $styleguideLanguage['twoLetterIsoCode']
                     ]
                 ));
-                $GLOBALS['TYPO3_REQUEST'] = $request;
             }
         }
 
         // Create view
-        $view = $this->createView('fluidStyleguide', 'Styleguide', $actionName);
+        $view = $this->container->get(StandaloneView::class);
+
+        $extbaseAttribute = new ExtbaseRequestParameters();
```

Comment:
The view is not properly initiated in TYPO3 v11.

`Tried resolving a template file for controller action "Standard->index" in format ".html", but none of the paths contained the expected template file (Standard/Index.html). The following paths were checked: /var/www/html/public/typo3conf/ext/fluid_styleguide/Resources/Private/Templates/`